tests/template.go

```diff
@@ -16,7 +16,9 @@ func (c Cookiecutter) CreateProjectFromGinTemplate(currentTemplatePath string) {
 	}
 
 	go func() {
+		defer GinkgoRecover()
 		defer stdin.Close()
```

Comment:
Could you explain more what this change does? 🙇

Reply:
Good catch. Before, I got this error when running tests:
![template_go_—_gin-templates](https://user-images.githubusercontent.com/7344405/220582207-87c866f3-2904-48e1-9132-a89bf39fc8e3.png)

But I realized that I did not install the `cookiecutter`, then it raised that error.
Removed in https://github.com/nimblehq/gin-templates/pull/96/commits/ac04768813185a7747041ca25864fbc158605bc3